pkg/question/question.go

```diff
@@ -1159,24 +1214,59 @@ func AskInstanceIds(h *ec2helper.EC2Helper, addedInstanceIds []string) (*string,
 	return &answer, err
 }
 
+func AskBootScriptConfirmation(h *ec2helper.EC2Helper) string {
+	stringOptions := []string{cli.ResponseYes, cli.ResponseNo}
+	optionsText := yesNoOption + "\n"
+	question := "Confirm to add a filepath to an instance boot script"
+
+	answer := AskQuestion(&AskQuestionInput{
+		QuestionString: question,
+		DefaultOption:  aws.String(cli.ResponseNo),
+		EC2Helper:      h,
+		Fns:            []CheckInput{ec2helper.ValidateFilepath},
+		OptionsString:  &optionsText,
+		StringOptions:  stringOptions,
+	})
+	return answer
```

Comment:
The extra variables make this code kinda noisy
```suggestion
	return AskQuestion(&AskQuestionInput{
		QuestionString: "Confirm to add a filepath to an instance boot script",
		DefaultOption:  aws.String(cli.ResponseNo),
		EC2Helper:      h,
		Fns:            []CheckInput{ec2helper.ValidateFilepath},
		OptionsString:  aws.String(yesNoOption + "\n"),
		StringOptions:  []string{cli.ResponseYes, cli.ResponseNo},
	})
```

Reply:
This is the format of pretty much all of the questions. Should they all be reformatted?